packages/login_client/lib/src/login_client.dart

```diff
@@ -87,8 +83,12 @@ class LoginClient extends http.BaseClient {
   Future<oauth2.Credentials?> get credentials => _credentialsStorage.read();
 
   /// Restores saved credentials from the credentials storage.
+  ///
+  /// You can provide a custom `tokenEndpoint` to override the one stored in
+  /// the credentials. This is useful when the token endpoint changes.
   Future<void> initialize() async {
```

Comment:
Let's add test that checks whether the credentials do have updated `tokenEndpoint` on initialization